src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/EmailDomainController.kt

```diff
@@ -1,70 +1,55 @@
 package uk.gov.justice.digital.hmpps.externalusersapi.resource
```

Comment:
Your endpoints are missing swagger docs

Reply:
If you're ok i'd prefer to break this out into a separate ticket which i'll pick up last as possible they may end up changing again?

Reply:
sure